src/pages/App.js

```diff
@@ -63,7 +64,32 @@ export default function App() {
       />
       <button type="submit" onClick={handleSubmit}>Search</button>
       <p>
-        {foods.map((elem) => <li>{elem.description}</li>)}
+        {foods.slice(10, 30).map((elem) => {
```

Comment:
This `slice` will get foods from the 10th index to the 29th. Is that the intended function of it?
If this line is intended to limit results, I think that would be better done in the backend so we aren't fetching unused results.

Reply:
I tried to limit result by just showing 5 results or 10 results but I already filter out two more issues which was:
1. Same name that user put in the search feature , I got rid of that because like I said if user type "chicken" the food description will show "chicken", "chicken" .
2. Also this API has lots of duplicate food items. So I got rid of that too.

Now I can't slice from 0 to 5 or 0 to 10 because It's going to show two to three result this is why I picked 10 to 30 where all the food has good description on average and most of them are unique. By doing slice from 10 to 30 I get 5-10 results on average.

Reply:
Gotcha. Like I said, I think limiting the search results may be better for the backend, but I suppose that can be done in a separate PR.